compose.yaml

```
@@ -0,0 +1,15 @@
+services:
+
+  backend:
+    build: .
+    container_name: backend
+    ports:
+      - "8000:8000"
```

```suggestion
      - "127.0.0.1:${HTTP_PORT:-8000}:8000"
```

Assim possibilita que a porta http seja customizável sem precisar alterar o docker-compose.yml caso alguém queira utilizar por exemplo na porta 80 para não ter de acrescentar uma porta na URL.

Prefixar a porta da máquina host com o IP da máquina host vai restringir o acesso a esta porta apenas ao IP 127.0.0.1, o que é interessante caso acessemos um wi-fi público para não ficarmos expondo a porta que a aplicação está utilizando. Neste caso não há muito risco pois é um site estático, mas de qualquer forma é sempre bom pensarmos em questões de segurança.